docs/build/step1.md

```diff
@@ -36,18 +35,19 @@
         * You do NOT need access to an Apple computer to update your phone iOS, troubleshoot or change Loop settings
 
 
-If you have access to a computer with MacOS 12.5 or newer, you can skip ahead to [Check Space Available](#check-the-space-available).
+If you have access to a computer with MacOS 13.0 or newer, you can skip ahead to [Check Space Available](#check-the-space-available).
 
 ## Compatible Versions
 
 With the [release of Loop 3](../version/releases.md#loop-3-compatibility):
 
-* Minium Mac operating system is Monterey, macOS 12.5 or higher
-* Mac operating system of Ventura, macOS 13.x is available and works fine to build Loop
+* Mac operating system of Ventura, macOS 13.x must be used if running on a phone with iOS 16.4 or newer
+* Minium Mac operating system is Monterey, macOS 12.5 or higher for phones running iOS 15.x or 16.3.1 or older
```

Comment:
📝 Typos: 
- ① Replace `Minium` with `Minimum`
- ② Replace `macOS` with `MacOS` for the corresponding tooltip to be triggered

Reply:
Thanks. This is a work in progress. I’ll fix this before merging.